docs/rfcs/0561-list-metadata-reuse.md

```diff
@@ -0,0 +1,210 @@
+- Proposal Name: `list_metadata_reuse`
+- Start Date: 2022-08-23
+- RFC PR: [datafuselabs/opendal#561](https://github.com/datafuselabs/opendal/pull/561)
+- Tracking Issue: None
```

Comment:
This RFC is accepted. Please create a tracking issue for this RFC and add the link here.